### DISCUSSION TITLE
Can be ported to Minetest

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [HbmMods/Hbm-s-Nuclear-Tech-GIT#629] [Q&A] [answered]
Can be ported to [Minetest](https://www.minetest.net/) ?
[Minetest mods](https://content.minetest.net/)

**Comment by TheForge129** [comment, marked as answer]
I don't think HBM is even going to justify this with an answer, so I'll do
it for him:

No.

It would require a complete rewrite of the entire mod in a different coding
language in a different engine.

Feel free to do it yourself though.

**Reply by Shibva** [reply]
I can agree with that logic, Even porting the mod to newer versions via forks is a lot of work; it makes you take into consideration the mad-lads that actually step up to the plate-like Drillgon did with the 1.12.2 fork
